environments/openshift/obs.jsonnet

```diff
@@ -758,6 +794,54 @@ local up = (import '../../components/up.libsonnet');
         name: 'JAEGER_PROXY_MEMORY_LIMITS',
         value: '200Mi',
       },
+      {
+        name: 'THANOS_STORE_MEMCACHED_IMAGE',
+        value: 'docker.io/memcached',
+      },
+      {
+        name: 'THANOS_STORE_MEMCACHED_IMAGE_TAG',
+        value: '1.6.3-alpine',
+      },
+      {
+        name: 'THANOS_STORE_MEMCACHED_EXPORTER_IMAGE',
+        value: 'docker.io/prom/memcached-exporter',
+      },
+      {
+        name: 'THANOS_STORE_MEMCACHED_EXPORTER_IMAGE_TAG',
+        value: 'v0.6.0',
+      },
+      {
+        name: 'THANOS_STORE_MEMCACHED_CPU_REQUEST',
+        value: '500m',
+      },
+      {
+        name: 'THANOS_STORE_MEMCACHED_CPU_LIMIT',
+        value: '3',
+      },
+      {
+        name: 'THANOS_STORE_MEMCACHED_MEMORY_REQUEST',
+        value: '1329Mi',
+      },
+      {
+        name: 'THANOS_STORE_MEMCACHED_MEMORY_LIMIT',
+        value: '1844Mi',
```

Comment:
Why so weird number? (:

Reply:
Just copy-paste from Telemeter, I haven't given much thought on the limits yet. Any suggestions?

Reply:
Anything just simple (:  We have 20GB to spare so let's start with ... 5?

Reply:
Should be good now.